FFmpegInterop/SubtitleProvider.h

```diff
@@ -70,7 +72,7 @@ namespace FFmpegInterop
 				TimeSpan duration;
 				bool isDurationFixed = false;
 
-				position.Duration = LONGLONG(av_q2d(m_pAvStream->time_base) * 10000000 * packet->pts) - m_startOffset;
+				position.Duration = LONGLONG(av_q2d(m_pAvStream->time_base) * 10000000 * packet->pts) - m_startOffset + AdditionalStreamSampleOffset.Duration;
```

Comment:
I know it would blow things up, but we could fix cues in the track here as well, like in the external subs provider. Maybe it is not really needed, not sure...